src/views/network/Network.vue

```diff
@@ -195,6 +217,12 @@
                 this.stickyBottom()
             }
         },
+        created () {
+            window.addEventListener('keydown', key => this.show_hide(key))
+        },
+        destroyed () {
+            window.removeEventListener('keydown', key => this.show_hide(key))
+        },
         mounted () {
             this.nextItem = this.nextItem.bind(this)
```

Comment:
As far as I've tested, this is not necessary, you can remove it if you want.
The idea was to have `this` available inside the function even if it's not called by the class itself.
But for some reason it is working even without this line 🤷🏻‍♂️, so you don't have to do the same.